package.json

```diff
@@ -53,9 +53,12 @@
     "debug": "^4.3.2",
     "file-type": "^16.5.3",
     "fs-extra": "^10.0.0",
+    "gm": "^1.23.1",
     "graph-data-structure": "^1.15.0",
     "image-size": "^1.0.0",
+    "mime": "^3.0.0",
     "mime-types": "^2.1.33",
+    "util": "^0.12.4",
```

Comment:
Not needed: https://www.npmjs.com/package/util => `You usually do not have to install util yourself. If your code runs in Node.js, util is built in. If your code runs in the browser, bundlers like browserify or webpack also include the util module.`

Reply:
removed